apluslms_roman/utils/path_mapping.py

```diff
@@ -0,0 +1,81 @@
+from pathlib import PurePosixPath
+import json
+import logging
+import re
+from os import environ
+
+logger = logging.getLogger(__name__)
+
+
+def get_host_path(original, mapping):
+    ret = original
+    orig_path = PurePosixPath(original)
+    for k, v in mapping.items():
+        try:
+            logger.debug("Mapping:{}:{}".format(k, v))
+            relative_path = orig_path.relative_to(k)
+            ret = PurePosixPath(v).joinpath(relative_path)
+            return str(ret)
+        except ValueError:
+            logger.debug("Error when composing new path!")
+            pass
+    return str(ret)
+
+
+def get_pair_form_env(key, json_str, read_key=None):
+    if key == read_key:
+        try:
+            ret = json.loads(json_str)
+            if isinstance(ret, dict):
+                return ret
+        except json.decoder.JSONDecodeError:
+            logger.debug("Error, check your json string")
+    return json_str
+
+
+def env_process_key(key, prefix):
+    return
+
+
+def env_value_to_dict(json_str):
+    if re.match(r'^(?:["[{]|(?:-?[1-9]\d*(?:\.\d+)?|null|true|false)$)', json_str):
+        try:
+            ret = json.loads(json_str)
+            if isinstance(ret, dict):
```

Comment:
`true` is valid json and should be passed as `True`. Thus, this if statement is not needed.